modules/gis/src/lib/flat-geojson-to-binary.ts

```diff
@@ -440,7 +445,7 @@ function triangulatePolygon(
   // Indices returned by triangulation are relative to start
   // of polygon, so we need to offset
   for (let t = 0, tl = triangles.length; t < tl; ++t) {
-    polygons.triangles.push(startPosition + triangles[t]);
+    polygons.triangles!.push(startPosition + triangles[t]);
```

Comment:
Nit: I am not a fan of the exclamation mark to override typings. I find it too subtle. Can we use an if statement instead?

Reply:
Avoided by moving `polygon.triangles` check into function body